app/app.go

```diff
@@ -109,6 +110,7 @@ const (
 )
 
 var _ servertypes.Application = (*App)(nil)
+var _ ibctesting.TestingApp = (*App)(nil)
```

Comment:
A few comments in this file claim that a method implements the TestingApp interface so I think we should enforce that via a compile-time check.